package.json

```diff
@@ -27,7 +27,7 @@
     "bin/image-size.js"
   ],
   "engines": {
-    "node": ">=18.18.0"
+    "node": ">=18.4.0"
```

Comment:
This change should be fine, but I'm curious why this change was needed. 
This package can theoretically work even on nodejs 12 with a few minor changes, but I like to use the very little leverage I have as a package maintainer to push people to upgrade their runtimes, and for that reason I'd like to actually soon bump this to `>=v18.19.1` instead.

Reply:
It's not needed, this would allow Astro to use the package directly instead of patching/vendoring but that is not neccessary, but I will remove it from this PR